### ISSUE TITLE
New theme is less fun, less readable and less usable. Revert plz. xoxo

### ISSUE THREAD

**Issue by oatsandsugar** [cornelltech/unicornucopia-issue-tracker#86]

**Comment by everydayslang** [comment]
Hey sorry, we really didn't mean to turn down the fun. A lot's changed with this update (mentions, better sorting, personal stats, etc). What particular pieces of fun are you finding hard to live without?

**Comment by oatsandsugar** [comment]
No need to apologise, iteration is critical!

The less colourful UI, the smaller font and the increased ideas per page
make it less approachable. I also find putting new ideas in less fun
because they look smaller haha. Would love to chat

**Comment by everydayslang** [comment]
Yeah, would you be available tomorrow afternoon? We're up on the 8th floor.

**Comment by conscho** [comment]
Second that. Also:
- the font is serif which does not look good. Please keep sans serif.
- The icons in the upper left corner are not very attractive nor self explanatory. Please include tooltips.

**Comment by oatsandsugar** [comment]
Agreed on the font, serifs are not #AbsolutelyModern.

Also bring the colours and the white background back, makes the whole app
happier.
